.github/workflows/test.yml

```diff
@@ -57,12 +57,8 @@ jobs:
       run: yarn test:lint
       working-directory: Meadowlark-js
 
-    - name: Build
-      run: yarn build
-      working-directory: Meadowlark-js
```

Comment:
I did some more poking around at this issue. Why is it passing in GitHub Actions without the build, but failing on my computer and on my development VM?

Experiment:

1. Ran `git clean -xdf`.
2. Looked at coverage report archived on a PR to find an uncovered module (`NoDocumentStorePlugin`).
3. Introduced a "compiler" error in that uncovered module.
4. Run `yarn test:unit`.
5. Successfully errored out on that module.
6. Revert the change.
7. Run `yarn test:unit` again ✔️ 
8. Run `yarn test:system` ❌ - there is a timing problem on my computer, where some updates fail to complete. This happened previously too, before this experiment.

Interestingly, the system tests also fail on my VM, but they don't fail in GitHub Actions. So it is not just my messed up computer

Reply:
I was able to resolve the timing problem with addition of `--runInBand` on the jest command. Git clean resolved the yarn build issue. 

Since my breaking change to `NoDocumentStorePlugin` was caught by the test process, I'm _not_ going to restore `yarn build` to the workflow. My original concern is covered already.